pebble/common.py

```diff
@@ -134,7 +137,7 @@ def launch_thread(function, *args , **kwargs):
 
 
 def launch_process(function, *args, **kwargs):
-    name = kwargs.pop('name', None)
+    name = kwargs.pop("name", None)
```

Comment:
I chose to `pop` because IMO the wrapper doesn't need the name parameter, maybe `get` would be better. Tell me what you think.